### DISCUSSION TITLE
Howdy as Hello style authentication for Linux

### DISCUSSION THREAD

**Discussion by benoe77** [jrandiny/yoga-slim7-ubuntu#29] [Ideas]
Another tweak that is working great on this notebook is to get Howdy face recognition for login and sudo:

First, you need to enable ir emmitter with the following tool:

[Linux Enable IR Emitter](https://github.com/EmixamPP/linux-enable-ir-emitter)

Then you can install and setup Howdy to use the ir camera:

[Howdy](https://github.com/boltgolt/howdy)

On my Yoga Slim 7 ir camera is /dev/video2

**Comment by jrandiny** [comment]
Hi, can you confirm that the IR emitter really works? The last time I test there's a problem with the main emitter not emitting, only the secondary smaller emitter is working. I have also tested the repo that you mention with no success. Can you confirm that the main emitter is working (the main emitter is the one on the left of the main camera)?

**Reply by benoe77** [reply]
It seems to be working for me: 
![Képernyőkép 2021-05-06 191706](https://user-images.githubusercontent.com/8297758/117339218-c69ad100-ae9f-11eb-94ec-e5dcfca4e538.png)
